### PR TITLE
added right menu

### LINKED ISSUE #35: Implement right menu bar

### PR DESCRIPTION
**Pull request by romaGri**
closed #35

### CODE REVIEW

File: ecommers/lib/ui/pages/search_page.dart [hunk 1]
```diff
@@ -1,8 +1,9 @@
+import 'package:ecommers/ui/widgets/right_menu_bar/index.dart';
```
Comment (Platton96) [inline ecommers/lib/ui/pages/search_page.dart]: removed

File: ecommers/lib/ui/pages/search_page.dart [hunk 2]
```diff
@@ -1,8 +1,9 @@
...
 import 'package:flutter/material.dart';
 
 class SearchPage extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
-    return const Text('Search');
-  }
+    return RightMenuWidget();
+    }
```
[the remark this answers is not in the document]
Comment (Platton96) [reply ecommers/lib/ui/pages/search_page.dart]: reset changes

File: ecommers/lib/ui/widgets/right_menu_bar/right_menu_item.dart [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+            const SizedBox(width: 25.0),
+            Expanded(
+              child: Text(
+                itemModel.title,
+                style: Theme.of(context).textTheme.subtitle1,
+              ),
+            ),
+            Align(
+              alignment: Alignment.centerLeft,
+              child: Container(child: _subTitleWidget(context, itemModel)),
```
[the remark this answers is not in the document]
Comment (Platton96) [reply ecommers/lib/ui/widgets/right_menu_bar/right_menu_item.dart]: why do you use height as property?

File: ecommers/lib/ui/widgets/right_menu_bar/right_menu_item.dart [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+          .copyWith(fontWeight: FontWeight.w200),
+    );
+  }
+  if (itemModel is PriceModel) {
+    final priceModel = itemModel;
+    final _minPrice = priceModel.minPrice;
+    final _maxPrice = priceModel.maxPrice;
+
+    return Text(
+      '\$ $_minPrice - \$ $_maxPrice',
```
Comment (Platton96) [inline ecommers/lib/ui/widgets/right_menu_bar/right_menu_item.dart]: use formatter for price

File: ecommers/lib/ui/widgets/right_menu_bar/right_menu_item.dart [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+      style: Theme.of(context)
+          .textTheme
+          .subtitle2
+          .copyWith(fontWeight: FontWeight.w200),
+    );
+  }
+  return const Text('null');
```
Comment (Platton96) [inline ecommers/lib/ui/widgets/right_menu_bar/right_menu_item.dart]: what is iText('null') ?

File: ecommers/lib/ui/widgets/right_menu_bar/right_menu_item.dart (hunk 4 above)
Comment (fryette) [reply ecommers/lib/ui/widgets/right_menu_bar/right_menu_item.dart]: check Intl package for formatting

File: ecommers/lib/ui/widgets/right_menu_bar/models/colors_model.dart [hunk 6]
```diff
@@ -0,0 +1,36 @@
+import 'package:ecommers/ui/widgets/right_menu_bar/models/index.dart';
+import 'package:flutter/widgets.dart';
+
+import '../../../decorations/dimens/index.dart';
```
Comment (fryette) [inline ecommers/lib/ui/widgets/right_menu_bar/models/colors_model.dart]: avoid using import like that

**Pull request closed without merging**